File: C++/Programming Basics with C++/07. Exercise - Nested Conditional Statement/simple_task/Volleyball.cpp
#include <iostream>
#include <string>
#include <algorithm>
#include <cmath>

using namespace std;

int main() {

    string typeOfYear;
    int holidays;
    int weekendTripCount;
    cin >> typeOfYear >> holidays >> weekendTripCount;
    transform(typeOfYear.begin(),
              typeOfYear.end(),
              typeOfYear.begin(),
              ::tolower);

    double numberSportingDays = 0;
    if (typeOfYear == "normal") {
        numberSportingDays = weekendTripCount + (48 - weekendTripCount) * 3 / 4.0 + holidays * 2 / 3.0;
        cout << floor(numberSportingDays) << endl;
    } else if (typeOfYear == "leap") {
        numberSportingDays = (weekendTripCount + (48 - weekendTripCount) * 3 / 4.0 + holidays * 2 / 3.0) * 1.15;
        cout << floor(numberSportingDays)<<endl;
    }
    return 0;
}

File: C#/Programming Basics with C#/07. Nested Conditional Statements - Exercise/03. Volleyball/Volleyball.cs
using System;

class Volleyball
{
    static void Main()
    {
        //Console.WriteLine(Math.Floor(-5.2544));//-6
        //Console.WriteLine(Math.Truncate(-5.2544));//-5
        //Console.WriteLine(Math.Floor(5.2544));//5
        //Console.WriteLine(Math.Truncate(5.2544));//5

        string typeOfYear = Console.ReadLine().ToLower();
        int holidays = int.Parse(Console.ReadLine());
        int wikendTripCount = int.Parse(Console.ReadLine());

        double numberSportingDays = 0;
        if (typeOfYear == "normal")
        {
            numberSportingDays = wikendTripCount + (48 - wikendTripCount) * 3 / 4.0 + holidays * 2 / 3.0;
            Console.WriteLine(Math.Floor(numberSportingDays));
        }
        else if (typeOfYear == "leap")
        {
            numberSportingDays = (wikendTripCount + (48 - wikendTripCount) * 3 / 4.0 + holidays * 2 / 3.0) * 1.15;
            Console.WriteLine(Math.Floor(numberSportingDays));
        }
    }
}